frontend/packages/console-app/src/components/nodes/NodeLogs.tsx

```diff
@@ -260,9 +260,11 @@ const NodeLogs: React.FC<NodeLogsProps> = ({ obj: node }) => {
           const links = !isWindows
             ? doc.querySelectorAll('a[href^="audit"]')
             : doc.querySelectorAll('a');
-          const filenames = [];
+          const filenames: string[] = [];
           for (const link of links) {
-            filenames.push(link.textContent);
+            if (link.textContent) {
+              filenames.push(link.textContent);
+            }
           }
```

Comment:
this field should always exist (as even empty elements have a textContent of an empty string). [textContent is only null if the node is a document or doctype](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent), which we are not selecting per the `querySelectorAll`..

```suggestion
            filenames.push(link.textContent as string);
          }
```